doc/src/build/validator-node.md

```diff
@@ -2,8 +2,117 @@
+
+The staking pool is implemented in a system-level smart contract (staking_pool.move) and is part of the Sui Framework.
+
+### User staking and withdrawals
+
+When users stake SUI tokens, these SUI objects are wrapped into `StakedSUI` objects. The calculation to determine each user’s relative ownership of the staking pool is done directly with the timestamp of the `StakedSUI` object (which determines the moment at which the deposit took place) and the change in the exchange rates between the deposit epoch and the withdrawal epoch. Each staking pool’s data structure contains a time series with that pool’s exchange rates. These exchange rates can be used to determine the withdrawals of any of the pool’s stakers.
+
+Stake withdrawals are processed immediately with the exchange rate prevailing at the previous epoch’s exchange rate. Withdrawals do not have to wait for the current epoch to close. Withdrawals include both the original stake the user deposited and all the stake rewards accumulated up to the previous epoch. Stakers do not earn the rewards accruing to their stake during the epoch at which they withdraw. Since there is no way to know how many stake rewards will be accumulated during the current epoch until the epoch closes, these cannot be included in the withdrawal. Hence, any user can withdraw their stake immediately and receive:
+
+*SUI withdrawn at E’ = ( SUI deposited at E ) * ( Exchange Rate at E’-1 / Exchange Rate at E )*
+
+## Validator Pool exchange rate
+
+The exchange rate for each validator pool is calculated at each epoch boundary as follows:
+
+*Exchange Rate at E+1 = ( 1 + ( Third-Party Staker Rewards at E / Third-Party Stake at E ) ) *( Exchange Rate at E )*
+
+The distinction between third-party owned vs validator-owned rewards and stake is relevant in that validators earn commissions on the staking pool’s tokens but third-party stakers do not. This accounting enables Sui to keep track of the rewards accrued by both validators and third-party token holders using a single global exchange rate table. 
+
+### Find the exchange rate
+
+Each epoch change emits a 0x2::validator_set::ValidatorEpochInfo event per validator with the exchange rate information. You can use the Events API to query events.
+
+## Staking rewards
+
+Within a given validator staking pool, all stakers receive the same proportion of rewards through the pool’s exchange rate appreciation. In addition, since validators earn commissions over the stake they manage, validators receive additional `StakedSUI` objects at the end of each epoch in proportion to the amount of commissions earned by their staking pool.
```

Comment:
```suggestion
Within a given validator staking pool, all stakers receive the same proportion of rewards through the pool’s exchange rate appreciation. In addition, since validators earn commissions over the stake they manage, validators receive additional `StakedSUI` objects at the end of each epoch in proportion to the amount of commissions their staking pool earns.
```